core/scripts/sql/texera_ddl.sql

```diff
@@ -15,18 +15,29 @@
 -- specific language governing permissions and limitations
 -- under the License.
 
+-- ============================================
```

Comment:
Can you also sync this change to `texera_ddl.sql` under the `deployment` folder?

Reply:
Is that necessary? We don't run test cases on deployments.

Reply:
Maintaining two DDL files for every change is a significant burden for developers. We should address issue #3823.

Reply:
Sure, I will solve #3823 seperately. This change is not needed in this PR.